viscy/scripts/network_diagram.py

```diff
@@ -3,32 +3,96 @@
 
 from viscy.light.engine import VSUNet
 
-# %%
+# %% 2D UNet
 model = VSUNet(
+    architecture="2D",
+    model_config={
+        "in_channels": 2,
+        "out_channels": 1,
+    },
+)
+
+model_graph = draw_graph(
+    model,
+    model.example_input_array,
+    graph_name="2D UNet",
+    roll=True,
```

Comment:
"deconv" value for this key leads to an error during graph construction. 

```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
File ~/conda/envs/04_image_translation/lib/python3.10/site-packages/torchview/torchview.py:256, in forward_prop(model, x, device, model_graph, mode, **kwargs)
    255 if isinstance(x, (list, tuple)):
--> 256     _ = model.to(device)(*x, **kwargs)
    257 elif isinstance(x, Mapping):

File ~/conda/envs/04_image_translation/lib/python3.10/site-packages/torchview/recorder_tensor.py:146, in module_forward_wrapper.._module_forward_wrapper(mod, *args, **kwargs)
    144 # TODO: check if output contains RecorderTensor
    145 # this seems not to be necessary so far
--> 146 out = _orig_module_forward(mod, *args, **kwargs)
    148 model_graph.context_tracker['current_depth'] = cur_depth

File ~/conda/envs/04_image_translation/lib/python3.10/site-packages/torch/nn/modules/module.py:1501, in Module._call_impl(self, *args, **kwargs)
   1498 if not (self._backward_hooks or self._backward_pre_hooks or self._forward_hooks or self._forward_pre_hooks
   1499         or _global_backward_pre_hooks or _global_backward_hooks
   1500         or _global_forward_hooks or _global_forward_pre_hooks):
-> 1501     return forward_call(*args, **kwargs)
   1502 # Do not call functions when jit is used

File ~/code/viscy/viscy/light/engine.py:215, in VSUNet.forward(self, x)
    214 def forward(self, x) -> torch.Tensor:
--> 215     return self.model(x)

File ~/conda/envs/04_image_translation/lib/python3.10/site-packages/torchview/recorder_tensor.py:146, in module_forward_wrapper.._module_forward_wrapper(mod, *args, **kwargs)
...
    266     ) from e
    267 finally:
    268     model.train(saved_model_mode)
```

Reply:
I was planning to deprecate that option. Do you think there is a value in keeping (and fixing) it?